sys/net/gnrc/netif/init_devs/auto_init_esp_ieee802154.c

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2025 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ */
+
+/*
+ * @ingroup sys_auto_init_gnrc_netif
+ * @{
+ *
+ * @file
+ * @brief   Auto initialization for the ESP32x IEEE 802.15.4 network interface
+ *
+ * @author  Gunar Schorcht <gunar@schorcht.net>
+ */
+
+#include "log.h"
+#include "net/gnrc/netif/ieee802154.h"
+#include "include/init_devs.h"
+
+#include "esp_ieee802154_hal.h"
+#include "net/netdev/ieee802154_submac.h"
+
+/**
+ * @brief   Define stack parameters for the MAC layer thread
+ * @{
+ */
+#ifndef ESP_IEEE802154_MAC_STACKSIZE
+#define ESP_IEEE802154_MAC_STACKSIZE       (IEEE802154_STACKSIZE_DEFAULT)
+#endif
+
+#ifndef ESP_IEEE802154_MAC_PRIO
+#define ESP_IEEE802154_MAC_PRIO            (GNRC_NETIF_PRIO)
+#endif
+
+static char _esp_ieee802154_stack[ESP_IEEE802154_MAC_STACKSIZE];
+
```

Comment:
```suggestion
/** @} */

```

To close the group opened after the `@brief`. Doxygen wouldn't complain because it's a .c file, but nevertheless 🤷

Reply:
But then you'll have to remove the opening bracket 🤔

Reply:
Already done :wink: I will push some fixups soon.